src/app/pages/basics/TrafficLayerExample.js

```diff
@@ -0,0 +1,42 @@
+/* global google */
+import {
+  default as React,
```

Comment:
This PR is meant to go on top of #417 but I'm not sure it has been done correctly. His original PR created the TrafficLayer, but he didn't compile it into the code so I took it that extra step in order to get it working correctly.